input/OFCB/test1/STD900.info

```diff
@@ -0,0 +1,9 @@
+Reactor REP
+Fueltype MOX
```

Comment:
this is the basic CLASS file coming with a recipe base... it contains some information require by CLASS